pyproject.toml

```diff
@@ -61,8 +62,9 @@ addopts = """\
 django_find_project = false
 
 [tool.poetry.dependencies]
-python = ">=3.8"
+python = ">=3.7"
 django = ">=3.2"
+importlib-metadata = "6.7.0"
```

Comment:
We need this library for Python3.7 to avoid error :
```
ModuleNotFoundError: No module named 'importlib_metadata'
```
Triggered here : https://github.com/pytest-dev/pluggy/blob/main/src/pluggy/_manager.py#L31C12-L31C30

I don't know if there's a way to install it only for Python3.7?